src/firefox/index.js

```diff
@@ -17,6 +17,53 @@ import {default as defaultFirefoxConnector, REMOTE_PORT} from './remote';
 
 const log = createLogger(__filename);
 
+// Flow Types
+
+import type {DefaultFirefoxConnectorFn} from './remote';
+import type {ChildProcess} from 'child_process';
+import type {
+  PreferencesAppName,
+  PreferencesGetterFn,
+} from './preferences';
+
+import type {ExtensionManifest} from '../util/manifest';
+
+// Declare the needed 'fx-runner' module flow types.
+
+export type FirefoxRunnerParams = {
```

Comment:
I'd prefer to see `FirefoxRunnerParams`, `FirefoxRunnerResults`, and `FirefoxRunnerFn` grouped above the [run()](https://github.com/rpl/web-ext/blob/d5083f42002b2dcb5e72f8c886997d0035c1c0c7/src/firefox/index.js#L112) command since they only applies to `FirefoxRunOptions`.